patches/components-signin-internal-identity_manager-BUILD.gn.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/components/signin/internal/identity_manager/BUILD.gn b/components/signin/internal/identity_manager/BUILD.gn
+index 893483064888ae2045114f0fd852993d3158b707..f294ffad85e99883315b4676966f3fec1e0a22d7 100644
+--- a/components/signin/internal/identity_manager/BUILD.gn
++++ b/components/signin/internal/identity_manager/BUILD.gn
+@@ -3,6 +3,7 @@
+ # found in the LICENSE file.
+ 
+ import("//components/signin/features.gni")
++import("//brave/components/signin/internal/identity_manager/sources.gni")
```

Comment:
including through src/brave/build/config/brave_build.gni gives build errors for Chromium test targets like 
```
ERROR at //components/signin/internal/identity_manager/BUILD.gn:130:14: Undefined identifier
  sources += brave_components_signin_internal_identity_manager_sources
             ^--------------------------------------------------------
See //components/BUILD.gn:146:5: which caused the file to be included.
    "//components/signin/internal/identity_manager:unit_tests",
    ^---------------------------------------------------------

```